etc/systemd/system-generators/zfs-mount-generator.in

```diff
@@ -104,8 +104,13 @@ process_line() {
       fi
 
       # Generate the key-load .service unit
-      cat > "${dest_norm}/${keyloadunit}" << EOF
-# Automatically generated by zfs-mount-generator
+      #
+      # Note: It is tempting to use a `<<EOF` style here-document for this, but
+      #   bash requires a writable /tmp or $TMPDIR for that. This is not always
+      #   available early during boot
```

Comment:
nit: add a period at the end of this sentence

Reply:
Here you go :)